apps/api/src/app/workflow-overrides/workflow-overrides.controller.ts

```diff
@@ -187,16 +188,13 @@ export class WorkflowOverridesController {
     summary: 'Delete workflow override',
   })
   @ExternalApiAccessible()
-  deleteWorkflowOverride(
-    @UserSession() user: IJwtPayload,
-    @Param('workflowOverrideId') workflowOverrideId: string
-  ): Promise<boolean> {
+  deleteWorkflowOverride(@UserSession() user: IJwtPayload, @Param('overrideId') overrideId: string): Promise<boolean> {
     return this.deleteWorkflowOverrideUsecase.execute(
       DeleteWorkflowOverrideCommand.create({
         organizationId: user.organizationId,
         environmentId: user.environmentId,
         userId: user._id,
-        _id: workflowOverrideId,
+        _id: overrideId,
```

Comment:
This is an example of a misconfigured endpoint that was caught by Spectral validation, and is now fixed to be consistent with other endpoint param names.

See the Spectral action CI error here: https://github.com/novuhq/novu/actions/runs/7114141317/job/19367636093#step:6:83